SpatialGDKEditorCommandlet/Source/Private/Commandlets/SchemaCommandlet.cpp

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SchemaCommandlet.h"
+#include "SpatialGDKEditorCommandletPluginPrivate.h"
+#include "SpatialGDKEditor.h"
+
+USchemaCommandlet::USchemaCommandlet()
+{
+	//TODO: HelpDescription = TEXT("Generate Schema and/or Snapshots");

```

Comment:
If this is meant for the follow up PR, please add a ticket and reference it here. Never mind if you intend to do this before this PR goes in.